core/store/vrf_key_store_test.go

```diff
@@ -39,7 +39,7 @@ func vrfVerifier() *solidity_vrf_verifier_wrapper.VRFTestHelper {
 	return verifier
 }
 
-var phrase = "englebert humperdinck is the greatest musician of all time"
+var phrase = "engelbert humperdinck is the greatest musician of all time"
```

Comment:
You've reduced the entropy of this super-secret password by dozens of bits!

Reply:
:laughing: